ingest-app/src/cmr/ingest/services/messages.clj

```diff
@@ -90,6 +90,14 @@
           (:Type related-url)
           (:Subtype related-url)))
 
+(defn url-content-type-type-subtype-not-matching-kms-keywords
+  "Error msg when URL Content Type, Type, and Subtype do not match a valid set in KMS."
+  [url]
+  (format "URL Content Type, Type, and Subtype [%s>%s>%s] are not a valid set together."
+          (:URLContentType url)
+          (:Type url)
+          (:Subtype url)))
```

Comment:
If any of these are nil this will substitute "null" into the string.  I guess that is ok as long as its intended.

Reply:
I would put an if here to see if Subtype is nil and depending on the outcome do the format statement accordingly
```
(if subtype
  (format "URL Content Type, Type, and Subtype [%s>%s>%s] are not a valid set together."
               (:URLContentType url)
               (:Type url)
              (:Subtype url))
  (format "URL Content Type, Type, and Subtype [%s>%s] are not a valid set together."
               (:URLContentType url)
               (:Type url)))
```

Reply:
Thanks for your feedbacks.  But looking at the kms rucontenttype.csv file, you can see all the URLContentType, Type and Subtypes are there, when Subtypes are nil, it's just an empty string "".  So, we're comparing them all. I think it's clearer to show which three values are not valid, rather than just two sometimes. It would be confusing to me.

Reply:
I can change it to (format "URL Content Type, Type, and Subtype [%s>%s>%s] are not a valid set together."
           (:URLContentType url)
           (:Type url)
           (get url :Subtype ""))

Reply:
Sounds good.

Reply:
Actually, RelatedURL's message is the same way. I think I'd better keep it consistent with RelatedURL's message since they compare the same thing. NULL is probably more clear than just an empty nothing there.